### DISCUSSION TITLE
Tested versions

### DISCUSSION THREAD

**Discussion by toyorg** [adi961/mib2-android-auto-vc#2] [General]
Works on `MHI2_ER_VWG13_K4525`. Great job!

**Comment by adi961** [comment]
thanks for the report!

**Comment by m4k4r0v89** [comment]
Not applicable (due to lack of a separate MMI display) for any variation of the following firmwares:
-MHI2_ER_AU43x
-MHI2_ER_AUG24
-MHI2_ER_AU62x
Great research done!

**Reply by adi961** [reply]
thank you!
